src/resources/locales/index.ts

```diff
@@ -12,6 +12,7 @@ import ru from "./ru.json";
 import uk from "./uk.json";
 import zhCN from "./zh-CN.json";
 import zhTW from "./zh-TW.json";
+import ptBR from "./pt-BR.json";
```

Comment:
It looks like the linter is complaining that these aren't in alphabetical order 🙄 
Could you put the ptBr one in between "pl" and "ru"?